VoxelSniperForge/src/main/java/com/github/kevindagame/voxelsniperforge/VoxelSniperForge.java

```diff
@@ -210,12 +220,27 @@ public List<VoxelEntityType> getEntityTypes() {
     @Nullable
     @Override
     public VoxelTreeType getTreeType(String namespace, String key) {
-        return ForgeRegistries.TREE_DECORATOR_TYPES.containsKey(new ResourceLocation(namespace, key)) ? new VoxelTreeType(namespace, key) : null;
+        return featureRegistry.get(new ResourceLocation(namespace, key)) != null ? new VoxelTreeType(namespace, key) : null;
+    }
+
+    @Override
+    @NotNull
+    public VoxelTreeType getDefaultTreeType() {
+        return new VoxelTreeType("minecraft", "oak");
```

Comment:
I don't like that you moved this hardcoded stuff in the main class. I think it's better to just return new VoxelTreeType wherever you call it

Reply:
The name is different between forge and spigot

Reply:
nvm, I realized why you did it

Reply:
nvm, I realized why you did it